### PR TITLE
[ MB-15434 ] Include ppm shipments in weight calculations

### PR DESCRIPTION
**Pull request by mmayo-truss**
## Summary
[This ticket](https://dp3.atlassian.net/browse/MB-15175) modified some existing weight calculation functions to include PPM shipment weights as part of the calculation. PPM shipment net weight is determined by taking the difference between the full and empty weight tickets, so retrieval of the weight tickets is a necessary part of this modified calculation.

In this PR we are modifying the queries on three other pages which calculate the total weight requested in order to include the weight tickets. This will allow the PPM net weights to be included in the total weight requested. 

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the UI locally.

```sh
make client_run
```

##### Terminal 2

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the office app
2. Login as a TOO
3. Select a Move with PPM shipments (XSWT01 or XSWT02 are both good ones for this)
4. Go to the Move Task Order tab.
5. There you should see the PPM net weights included in the Move Weight Total weight display box. 
6. Go to the Payment Requests tab
7. There you should see the PPM net weights are included in the weight requested section (note it is not included in the Total billable weight)
8. Click the review weights button, and you will go to the Review Weights, Edit Max Billable Weights page
9. There you should see the PPM net weights are included in the weight requested section (note it is not included in the Total billable weight)

## Screenshots
MTO page as a TOO includes ppm net weights in move weight total:
image

Payment Requests Page shows the the weight requested includes ppm net weights:
image

Review Weights page has the weight requested updated to include PPM shipments: 
image

### CODE REVIEW

File: src/hooks/custom.js [hunk 1]
```diff
@@ -24,6 +32,14 @@ export const useCalculatedTotalBillableWeight = (mtoShipments) => {
   }, [mtoShipments]);
 };
 
+/**
+ * This function calculates the weight requested of a move,
```
Comment (michaelvaldes-truss) [inline src/hooks/custom.js:36]: **nitpick:** we might want to call out here that ppm shipments are expected to have weight tickets attached to them

File: src/hooks/queries.js [hunk 2]
```diff
@@ -61,6 +61,42 @@ import {
 } from 'constants/queryKeys';
 import { PAGINATION_PAGE_DEFAULT, PAGINATION_PAGE_SIZE_DEFAULT } from 'constants/queues';
 
+/**
```
Comment (michaelvaldes-truss) [inline src/hooks/queries.js:64]: **praise:** nice reusable hook!

**Review by michaelvaldes-truss — APPROVED** [review]
lgtm! This cleanly addresses inconsistencies in weight calculations between the SC and TXO

File: src/hooks/custom.js (hunk 1 above)
Comment (mmayo-truss) [reply src/hooks/custom.js:36]: @michaelvaldes-truss Very good idea. I will incorporate this suggestion before I merge.

**Pull request merged**